packages/liveblocks-node/src/__tests__/client.test.ts

```diff
@@ -1164,4 +1164,48 @@ describe("client", () => {
       })
     ).resolves.toBeUndefined();
   });
+
+  test("should delete user's inbox notification", async () => {
+    const userId = "user1";
+    const inboxNotificationId = "in_123";
+
+    server.use(
+      http.delete(
+        `${DEFAULT_BASE_URL}/v2/users/:userId/inbox-notifications/:inboxNotificationId`,
+        () => {
+          return HttpResponse.json(undefined, { status: 204 });
+        }
+      )
+    );
+
+    const client = new Liveblocks({ secret: "sk_xxx" });
+
+    await expect(
+      client.deleteInboxNotification({
+        userId,
+        inboxNotificationId,
+      })
+    ).resolves.toBeUndefined();
+  });
+
+  test("should delete all user's inbox notifications", async () => {
+    const userId = "user1";
+
+    server.use(
+      http.delete(
+        `${DEFAULT_BASE_URL}/v2/users/:userId/inbox-notifications`,
+        () => {
+          return HttpResponse.json(undefined, { status: 204 });
+        }
+      )
+    );
+
+    const client = new Liveblocks({ secret: "sk_xxx" });
+
+    await expect(
+      client.deleteInboxNotifications({
+        userId,
+      })
+    ).resolves.toBeUndefined();
```

Comment:
maybe add a test to assert that it throws when the server returns something else than 2XX?